external/lfs.patch

```diff
@@ -1,9 +1,13 @@
-commit aef9047d67d08027afe0417d6c681bfdc4666f5e
-Author: rasapala <rafal.a.sapala@intel.com>
-Date:   Fri Mar 14 09:23:35 2025 +0100
-
-    Lfs
-
+diff --git a/.gitignore b/.gitignore
```

Comment:
could we make it a PR to gitlib2?

Reply:
We can try, we have some hardcoded things we use specifically for ovms and our windows compilation.
Lets test it and see if it works in all scenarios in demos and the I can prepare upstream patch for libgit.